src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -182,6 +184,25 @@ public void writeItemStackData(NBTTagCompound itemStack) {
         }
     }
 
+    /**
+     * Returns tier less tooltip key if MetaTileEntity has tiers (lv, mv, ...)
+     *
+     * @return tier less tooltip key or null (if does not have tiers)
+     */
+    public final String getTierlessTooltipKey() {
```

Comment:
bad implementation. Shouldn't do anything but return getMetaName() in base meta tile entity class at all.
It's functionality should be implemented in TieredMetaTileEntity, and not like that.